### PR TITLE
Simran changes

### PR DESCRIPTION
**Pull request by simran-kaurbrar**
Sticky Button added to Home and Search Page
SearchPage Card is updated now

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
-import App from 'src/App.jsx';
+import App from './App.jsx';
```
Comment (mgao08) [inline src/main.jsx:4]: I dont know why the absolute path does not work on your side, but on my side it works without problem. I think its better to change it back to the original absolute path.

File: src/pages/Search/Search.jsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import styles from "./Search.module.css";
 import React, { useState, useEffect } from "react";
 
 import SearchCard from "./components/SearchCard/SearchCard";
+import StickyButton from './components/stickybutton/stickybutton';
```
Comment (mgao08) [inline src/pages/Search/Search.jsx:6]: Don't forget to change your import path

**Review by mgao08 — CHANGES_REQUESTED** [review]

**Review by mgao08 — DISMISSED** [review]
Nice job on your first pull request 🎊 
But you might want to double check the Discord messages about which branch to merge into. As she mentioned the `main` branch is not in use anymore

**Review by mgao08 — DISMISSED** [review]
Changed base branch from `main` to `development` as requested.
Okay to merge as soon as merge conflicts are solved.

File: src/pages/Search/components/SearchCard/SearchCard.module.css [hunk 3]
```diff
@@ -1,4 +1,71 @@
...
+ @media screen and (max-width: 575px){
+   .card {
+      width: 100%;
+      margin-bottom: 1rem; 
+    }
+    .footer{
+      display:block;
+    }
+ }
+=======
```
Comment (mgao08) [inline src/pages/Search/components/SearchCard/SearchCard.module.css:68]: Please run your local file and solve all the errors before requesting others to review

**Review by mgao08 — CHANGES_REQUESTED** [review]

File: src/pages/Search/components/SearchCard/SearchCard.module.css (hunk 3 above)
Comment (simran-kaurbrar) [reply src/pages/Search/components/SearchCard/SearchCard.module.css]: i updated this one and now it is working on my local file as well.

**Review by mgao08 — APPROVED** [review]

**Pull request merged**